tests/sagemaker/test_deployment.py

```diff
@@ -1,875 +0,0 @@
-import os
```

Comment:
All of the coverage provided in this test suite for `mlflow.sagemaker.deploy() / delete()` is covered for `SageMakerDeploymentClient` in https://github.com/mlflow/mlflow/blob/master/tests/sagemaker/test_sagemaker_deployment_client.py